sycl/test-e2e/Complex/sycl_complex_marray_getters_test.cpp

```diff
@@ -0,0 +1,92 @@
+// RUN: %{build} -fsycl-device-code-split=per_kernel -o %t.out
```

Comment:
Is `-fsycl-device-code-split=per_kernel` necessary? If so, please add a comment why, otherwise remove it.